.github/workflows/destroy-frontend-preview.yml

```
@@ -0,0 +1,351 @@
+      target-user: ${{ steps.codeowner-check.outputs.target-user }}
+      amplify-app-name: ${{ steps.amplify-info.outputs.amplify-app-name }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Check code ownership
+        id: codeowner-check
+        uses: ./.github/actions/check-codeowner
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.7)

input "github-token" is not defined in action "Check Code Owner" defined at "./.github/actions/check-codeowner". available inputs are "base-ref", "mode"

(action)